apps/super-app/src/services/SupportServices.ts

```diff
@@ -1,10 +1,23 @@
 // TODO This is a stub for testing, eventually replace this with Strapi Calls
+
+import { Path } from '../routes/paths';
```

Comment:
Ah, I hadn't considered the problem of adding links to this page.. I think this is okay for now, but we have no good way for the actual Strapi service to know what the app internal constants are, so when we move to Strapi, we will have to drop this import. Let's leave a TODO to note that we need to figure this out later.

I'm quite certain that the backend shouldn't be able to tell the frontend what the route is, and we need to create each support detail page anyway, so and it doesn't make sense for the backend to tell the frontend what the list of support services is.

I'm not sure what the 'right' way forward is, here are some ideas:

- Keep the `getServiceCardAbouts()`, and teach the frontend to identify the route that should correspond to a service. (e.g. something like associate each support service entry with an 'id' and store a map of support service id to route in the frontend). Seems a little roundabout, but this maps well onto Strapi content types. I worry about the id being accidentally modified in Strapi and breaking the frontend, but maybe there are ways to make this less likely.
- Get rid of `getServiceCardAbouts()` and hardcode the list of support services in the frontend. Since the frontend should already know the list of services, it might make sense to just have the frontend reach out to the right backend routes. But if we do this, it probably makes the most sense for to have only one-off types for each Support service (e.g. `/pbs` and `/twc2` instead of `/support/1` and `/support/2`), and I don't know how this would map well onto Strapi.

WDYT? At any rate, we don't have to make a decision right now, it seems like this would change a lot based on how we experiment with Strapi.

Reply:
Let's do it as a TODO first?